extensions/amp-animation/amp-animation.md

```diff
@@ -299,6 +333,64 @@ can be reduced to an array of components. For instance:
 </amp-animation>
 ```
 
+
+### Animation composition
```

Comment:
Can this clarify further on the difference between setting `animation` as a reference vs. `animations`? Some confusion can come from the fact that they can be described top-level or nested one level down.